engines/decidim_hospitalet-surveys/app/commands/decidim_hospitalet/surveys/admin/create_survey_result.rb

```diff
@@ -89,6 +89,13 @@ def existing_user
           ).first
         end
 
+        def default_user
+          Decidim::User.where(
```

Comment:
Nope, users don't come when loading the schema, which is the recommended method to set up an installation, not migrating from scratch - that's why the schema file actually exists. I stand still with my approach xD. `find_or_create` is the way to go.